python/dgl/sampling/pinsage.py

```diff
@@ -96,6 +84,19 @@ def __init__(self, G, num_traversals, termination_prob,
 
     # pylint: disable=no-member
     def __call__(self, seed_nodes):
+        """
+        Parameters
+        ----------
+        seed_nodes : Tensor
```

Comment:
Any requirement for the device and dtype of the tensor?

Reply:
Added.